test/minter-suite-minters/DA/MinterDAExp/MinterDAExpV3.test.ts

```diff
@@ -0,0 +1,306 @@
+ * V3 core contract.
+ */
+for (const coreContractName of coreContractsToTest) {
+  describe(`MinterDAExpV3_${coreContractName}`, async function () {
+    beforeEach(async function () {
+      // standard accounts and constants
+      this.accounts = await getAccounts();
+      await assignDefaultConstants.call(this);
+      this.startingPrice = ethers.utils.parseEther("10");
+      this.higherPricePerTokenInWei = this.startingPrice.add(
+        ethers.utils.parseEther("0.1")
+      );
+      this.basePrice = ethers.utils.parseEther("0.05");
+      this.defaultHalfLife = ONE_HOUR / 2;
+      this.auctionStartTimeOffset = ONE_HOUR;
+
+      // deploy and configure minter filter and minter
+      ({
+        genArt721Core: this.genArt721Core,
+        minterFilter: this.minterFilter,
+        randomizer: this.randomizer,
+      } = await deployCoreWithMinterFilter.call(
+        this,
+        coreContractName,
+        "MinterFilterV1"
+      ));
+
+      this.targetMinterName = "MinterDAExpV3";
+      this.minter = await deployAndGet.call(this, this.targetMinterName, [
+        this.genArt721Core.address,
+        this.minterFilter.address,
+      ]);
+
+      await safeAddProject(
+        this.genArt721Core,
+        this.accounts.deployer,
+        this.accounts.artist.address
+      );
+
+      await this.genArt721Core
+        .connect(this.accounts.deployer)
+        .toggleProjectIsActive(this.projectZero);
+
+      await this.genArt721Core
+        .connect(this.accounts.artist)
+        .updateProjectMaxInvocations(this.projectZero, 15);
+
+      await this.genArt721Core
+        .connect(this.accounts.artist)
+        .toggleProjectIsPaused(this.projectZero);
+
+      await this.minterFilter
+        .connect(this.accounts.deployer)
+        .addApprovedMinter(this.minter.address);
+      await this.minterFilter
+        .connect(this.accounts.deployer)
+        .setMinterForProject(this.projectZero, this.minter.address);
+
+      if (!this.startTime) {
+        const blockNumber = await ethers.provider.getBlockNumber();
+        const block = await ethers.provider.getBlock(blockNumber);
+        this.startTime = block.timestamp;
+      }
+      this.startTime = this.startTime + ONE_DAY;
+
+      await ethers.provider.send("evm_mine", [this.startTime - ONE_MINUTE]);
+      await this.minter
+        .connect(this.accounts.deployer)
+        .resetAuctionDetails(this.projectZero);
+      await this.minter
+        .connect(this.accounts.artist)
+        .setAuctionDetails(
+          this.projectZero,
+          this.startTime + this.auctionStartTimeOffset,
+          this.defaultHalfLife,
+          this.startingPrice,
+          this.basePrice
+        );
+      await ethers.provider.send("evm_mine", [this.startTime]);
+    });
+
+    describe("common DAEXP tests", async function () {
+      await MinterDAExp_Common();
+    });
+
+    describe("common DA V1V2 tests", async function () {
+      await MinterDAV1V2_Common();
+    });
+
```

Comment:
We do want the same tests that were ran on V2 to be ran here.
Perhaps we refactor this in the future, but for now, I would probably rename `MinterDAV2_Common()` to `MinterDAV2V3_Common()`, then add the imports and run here, as well as update the names in `MinterDAExpV2.test.ts`.

```js
    describe("common DA V2 tests", async function () {
      await MinterDAV2_Common();
    });
```
This same pattern also should be updated in `MinterDALinV3.test.ts`, `MinterSetPriceV3.test.ts`, and `MinterSetPriceERC20V3.ts` 👍